app/services/idv/actions/verify_document_action.rb

```diff
@@ -65,8 +65,6 @@ def enqueue_job
         nil
       end
 
-      private
-
```

Comment:
Was this change meant to be included?

Reply:
Yes because private is in this file twice for some reason.